packages/react-instantsearch/src/core/InstantSearch.js

```diff
@@ -57,6 +57,9 @@ function validateNextProps(props, nextProps) {
  * @propType {string} appId - The Algolia application id.
  * @propType {string} apiKey - Your Algolia Search-Only API key.
  * @propType {string} indexName - The index in which to search.
+ * @propType {function=identity} configureSearchParameters - function to tweak the parameters sent to Algolia.
```

Comment:
That's a pretty good question, I have no idea if this concept is well known or if we should make it more approachable. The nice thing of using `identity` is that it convey perfectly what it means, there is no approximation about its meaning, on the contrary of using words. On the other hand, if you don't know this vocab,  you don't know it...

What do you think @mthuret? :)